MbDotNet/MountebankRequestProxy.cs

```diff
@@ -42,15 +42,34 @@ public void DeleteAllImposters()
 
         public void DeleteImposter(int port)
         {
-            var response = ExecuteDelete(string.Format("{0}/{1}", ImpostersResource, port));
-            HandleResponse(response, HttpStatusCode.OK, string.Format("Failed to delete the imposter with port {0}.", port));
+            var response = ExecuteDelete($"{ImpostersResource}/{port}");
+            HandleResponse(response, HttpStatusCode.OK, $"Failed to delete the imposter with port {port}.");
         }
 
         public void CreateImposter(Imposter imposter)
         {
             var json = JsonConvert.SerializeObject(imposter);
             var response = ExecutePost(ImpostersResource, json);
-            HandleResponse(response, HttpStatusCode.Created, string.Format("Failed to create the imposter with port {0} and protocol {1}.", imposter.Port, imposter.Protocol));
+            HandleResponse(response, HttpStatusCode.Created,
+                $"Failed to create the imposter with port {imposter.Port} and protocol {imposter.Protocol}.");
+        }
+
+        public RetrievedImposter GetImposter(int port)
+        {
+            var response = ExecuteGet($"{ImpostersResource}/{port}");
+            HandleResponse(response, HttpStatusCode.OK, $"Failed to retrieve imposter with port {port}");

```

Comment:
Perhaps it makes sense to handle a 404 when the imposter does not exist and return null instead of an exception. I could go either way though since mountebank technically does return an error.